shared/js/moz_intl.js

```diff
@@ -210,13 +534,13 @@ global.mozIntl = {
       const relativeFmtOptions = {
         unit: 'bestFit',
         style: style,
-        minUnit: 'minute',
+        minUnit: 'minute'
       };
 
       return {
-        format: function(time, maxDiff) {
-          maxDiff = maxDiff || 86400 * 10; // default = 10 days
-          const secDiff = (Date.now() - time) / 1000;
+        format: function (time, maxDiff) {
```

Comment:
This might be out of the scope of this bug, but would it be worthwhile to rename this method to something that would indicate that it's async and differentiate it from real `Intl`'s methods?  I'm slightly worried that it looks like `Intl`'s `format` but isn't really, because of a different return type.